src/components/tx/TxData.js

```diff
@@ -509,14 +505,19 @@ class TxData extends React.Component {
       ));
     };
 
-    const renderNewUiDivList = hashes => {
+    const renderTxListWithSpacer = hashes => {
+      const obj = hashes.map( h => <Link className="fs-14" to={`/transaction/${h}`}>{h}</Link> );
```

Comment:
Increased font size to 14px to improve usability. It was too small before.

Original
<img width="1710" alt="image" src="https://github.com/user-attachments/assets/8fd46495-cf60-4a3c-b468-d146b2d8ab2f" />


New
<img width="1710" alt="image" src="https://github.com/user-attachments/assets/14eb6a74-35d3-4f9b-a17a-81592b26a464" />